app/frontend/components/domains/authentication/login-screen.tsx

```diff
@@ -38,8 +38,8 @@ export const LoginScreen = ({}: ILoginScreenProps) => {
     <CenterContainer>
       <Flex direction="column" gap={6} w="full" p={10} border="solid 1px" borderColor="border.light" bg="greys.white">
         <Heading as="h1">{t("auth.login")}</Heading>
-        <form action="/api/auth/keycloak" method="post">
-          {/* @ts-ignore */}
+        {/* Disabling BCeID login pending IDIM approval */}
+        {/* <form action="/api/auth/keycloak" method="post">
```

Comment:
Does the form have closing tag to comment out?

Reply:
Ah github is just hiding it in the "show more"